README.md

```diff
@@ -73,6 +73,14 @@
             <sub><b>Сергей</b></sub>
         </a>
     </td></tr>
+<tr>
+    <td align="center">
+        <a href="https://github.com/ImgBotApp">
+            <img src="https://avatars.githubusercontent.com/u/31427850?v=4" width="100;" alt="ImgBotApp"/>
+            <br />
+            <sub><b>Imgbot</b></sub>
+        </a>
+    </td></tr>
 </table>
 <!-- readme: ImgBotApp/-,contributors -end -->
```

Comment:
@akhilmhdh hello!

`ImgBotApp/-` doesn't work at all.

I have tried your solution, and reordered, as you can see here. None is working.

Reply:
Hey @Searge Hope the issue is resolved. 

If you have any problem feel free to open an issue. Thank you for using my action 😃.